### PR TITLE
[ #158421931] create registration template

### PR DESCRIPTION
**Pull request by SnyderMbishai**
#### Introduction
This is a PR for the registration page.

#### Task description
It contains the registration fields that captures a users details.

#### Manual testing
Git clone the repo locally then open the file on a browser.

#### Screenshot
screenshot from 2018-06-19 21-21-32

### CODE REVIEW

File: UI/css/templates_css.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+
+body{
+    
+    background-image: url(../images/2.jpg);
+    padding: 10px;
+    margin: 0;
+    font-family: 'Comic Sans MS';
+    width: 100%;
+    height:100%;
+      }
```
Comment (AnthonyGW) [inline UI/css/templates_css.css]: Keep a consistent indentation

File: UI/css/templates_css.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#footer{
+  position: absolute;
+  bottom: 0px;
+  height: 40px;
+  left: 0px;
+  right: 0px;
+  overflow: hidden;
+  background-color: grey;
+  text-align: center;
+}
```
Comment (AnthonyGW) [inline UI/css/templates_css.css]: Insert a newline at the end of the file

File: UI/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <button>Register</button>
+            <button>Login</button>
+        </div>
+
+        <div id=footer>
+            <p>&copy; 2018 Mbishai</p>
+        </div>
+
+    </body>
+</html>
```
Comment (AnthonyGW) [inline UI/index.html]: Insert a newline at the end of the file

File: UI/css/templates_css.css (hunk 1 above)
Comment (SnyderMbishai) [reply UI/css/templates_css.css]: Noted.

File: UI/css/templates_css.css (hunk 2 above)
Comment (SnyderMbishai) [reply UI/css/templates_css.css]: I'm on it.

File: UI/index.html (hunk 3 above)
Comment (SnyderMbishai) [reply UI/index.html]: Done. Thanks for the feedback.

**Pull request merged**